public/squawk.html

```diff
@@ -0,0 +1,9 @@
+<html>
+<head>
+  <link rel="stylesheet" href="squawk.css">
```

Comment:
This page's entire purpose is to be loaded into the iframe so it clears the referrer. Don't load an unnecessary stylesheet here.